hello-world/hello_world_test.rb

```diff
@@ -1,8 +1,12 @@
 require 'minitest/autorun'
 require_relative 'hello_world'
 class HelloWorldTest < Minitest::Test
+  def test_no_strange_name
+    assert_equal 'Hello, !', HelloWorld.hello(''), 'Greeting given an empty string as a name'
+  end
+
   def test_no_name
-    assert_equal 'Hello, world!', HelloWorld.hello(''), 'Greeting given no name'
+    assert_equal 'Hello, world!', HelloWorld.hello, 'Greeting given an empty string as a name'
```

Comment:
The way I see it, this is a blank name: `hello("")`, whereas this is a nil name: `hello(nil)`, whereas this is no name: `hello`. Nil is sometimes nothing, sometimes it represents something. That's context-dependent, and we don't have context here, so it's hard to say whether the nil case actually means no name.

I think the failure messages should probably describe either what went wrong, but these are really used to identify which particular assertion failed. If we don't have trouble with context we should just delete the messages altogether.